pkgs/development/libraries/olm/default.nix

```diff
@@ -22,5 +22,8 @@ stdenv.mkDerivation rec {
   # requires optimisation but memory operations are compiled with -O0
   hardeningDisable = ["fortify"];
 
+  buildFlags = if stdenv.cc.isClang then [ "CC=clang" ] else null;
```

Comment:
makeFlags are used for build, check, and install. Using that is preferred.

Also, I think ```[ "CC=$(CC)" ]``` will work as well

Reply:
Thank you! Fixed.